themes/default/content/docs/guides/scim/onelogin.md

```diff
@@ -0,0 +1,153 @@
+---
+title: Configuring SCIM in OneLogin
+meta_desc: This page describes how to support SCIM 2.0 functionality between Pulumi and OneLogin.
+menu:
+    userguides:
+        parent: scim
+        weight: 1
+---
+
+This document outlines the steps required to help you configure automatic provisioning/deprovisioning of your users and groups in Pulumi using SCIM 2.0.
+
+Please note that some advanced SCIM features aren't supported yet. For more information, see [Known Limitations](#known-limitations).
+
+## Prerequisites
+
+* You must be an admin of your Pulumi organization.
+* (Optional, but highly recommended) You should have more than one admin for your Pulumi organization.
+
+## Configuring the OneLogin Application
+
+The first step is to create a new OneLogin Application for Pulumi SCIM:
+
+1. From the OneLogin Administration portal, go to the **Applications** page and click the **Add App** button.
+1. Search for `SCIM Provisioner with SAML (SCIM v2 Core)` and select it.
+
+1. Enter a _Display Name_ and optionally a logo. See [Pulumi Logos](https://www.pulumi.com/brand/#logos).
+1. Click **Save**.
```

Comment:
```suggestion
1. Select **Save**.
```

Reply:
Found a few other nefarious clicks that I changed to selects as well.